Tasks/GithubReleaseV0/task.json

```diff
@@ -47,11 +44,11 @@
             "label": "Action",
             "defaultValue": "Create",
             "required": true,
-            "helpMarkDown": "Select the type of release operation you want perform. This task can create, edit or discard a github release.",
+            "helpMarkDown": "Specify the type of release operation to perform. This task can create, edit, or discard a GitHub release.",
             "options": {
-                "Create": "Create",
-                "Edit": "Edit",
-                "Discard": "Discard"
+                "create": "Create",
```

Comment:
We used this value as $(action) in instanceNameFormat property below. So that task name can be modified according to action. For e.g. **Create GitHub Release** or **Edit GitHub Release**. But Lowercasing the options will lead to **create GitHub Release** or **edit GitHub Release**.

Reply:
@mdmdakbar, ah... I see what you mean.  To the user, these values are case-insensitive when used in their YAML pipelines.  Most GitHub users will use the typical YAML camel-casing and lowercase the the `action` value on their own, like this:

```yaml
- task: GithubRelease@0
  inputs:
    action: create
    gitHubConnection: myConnection
```

Since most people will lowercase it in their YAML (no matter what casing is used in task.yaml), what would you think about changing `instanceNameFormat` to have the action at the end, like this?
- GitHub release (create)
- GitHub release (edit)
- GitHub release (discard)